api/src/opentrons/ordered_set.py

```diff
@@ -41,6 +41,11 @@ def remove(self, element: _SetElementT) -> None:
         """
         del self._elements[element]
 
+    def remove_if_found(self, element: _SetElementT) -> None:
+        """Remove ``element`` from the set. No-op if not found."""
+        if element in self._elements:
+            del self._elements[element]
```

Comment:
It's a little request, but I think we should tend towards more of an "ask forgiveness" model

```python
```suggestion
        try:
            del self._elements[element]
        except KeyError:
            pass
```
```